apps/react/docs/docs/02-Installation/02-Option 2 Docker Compose Setup/01-docker-compose-filesoverview.mdx

```diff
@@ -0,0 +1,40 @@
+---
+id: docker-compose files overview
+title: 📦 Docker-Compose Files
+---
+
+# Docker-Compose Files Overview 📦
+
+:::info
```

Comment:
I got the feeling the "under development" and "For development/test Environments only" in different files are polutting the documentation. Should we put this on an index file at the installation folder?